roles/kubernetes/control-plane/templates/kubeadm-config.v1beta3.yaml.j2

```diff
@@ -269,14 +269,6 @@ apiServer:
     mountPath: {{ volume.mountPath }}
     readOnly: {{ volume.readOnly | d(not (volume.writable | d(false))) }}
 {% endfor %}
```

Comment:
Thanks @maxime1907 

Why the kubeadm template has changed, does it necessary :-)  ?

Reply:
The Kubeadm and kubespray's config are different :

- https://github.com/kubernetes/kubernetes/pull/59122/files#diff-3d47990365907fa7cc2268a831614f00d5cd620ee6c44071929a2034b1f74befR44
- https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubespray-defaults/defaults/main.yaml#L529

I'm not sure about It can be removed.
Would you please submit another PR to change the kubeadm config, and the reviewers can comment it. :-)